source/platform/renderers/OpenGL/OpenGL.cpp

```diff
@@ -44,7 +44,7 @@ namespace Renderer
 // See "Font Size in Pixels or Points" in "stb_truetype.h"
 static constexpr float c_pointsToPixels = 1.333f;
 // Arbitrary size chosen for texture cache target.
-static constexpr int c_texsz = 512;
+static constexpr int c_texsz = 800; // Texture size too small for wchar_t characters but stbtt_BakeFontBitmap crashes on larger sizes
```

Comment:
Does this have to be a power of 2?

Reply:
Not necessarily, this integer serves as both the width and height of the texture for a font. I increased the number because function stbtt_BakeFontBitmap() returns the highest Y value that was used on success or the negative of the index it failed at. It turns out that 512 is too small for the default font, all characters cannot be placed on the texture. 
![image](https://user-images.githubusercontent.com/4210885/42725091-210cdc42-874c-11e8-86e1-4dfe54f69d55.png)

![image](https://user-images.githubusercontent.com/4210885/42725112-72cadb7e-874c-11e8-8b0a-8f51ec3f0113.png)

Reply:
I was commenting on the crash. I wondered if texture size had to be power of 2 or a multiple of something, causing a platform specific issue.

Does FontStash use multiple cache textures? I.e. when one fills up, create another. I know Allegro does. Perhaps the "font module" needs to chain multiple pages instead of relying on a fixed size.